src/scan-handler.js

```diff
@@ -38,13 +38,13 @@ module.exports = function(rootPaths, regexSource, options, searchOptions = {}) {
   }, callback);
 };
 
-var processPaths = function(rootPath, paths) {
+unction processPaths(rootPath, paths) {
```

Comment:
```suggestion
function processPaths(rootPath, paths) {
```

Syntax error here, is likely causing some of our tests to fail. Since seemingly all tests on this file fails.